src/ADP.Portal.Core/Ado/Client/ADORestHttpClient.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.VisualStudio.Services.Common;
+using Microsoft.VisualStudio.Services.WebApi;
+
+namespace ADP.Portal.Core.Ado.Client
+{
+    public class AdoRestHttpClient : VssHttpClientBase
+    {
+        private readonly Uri baseUrl;
+
+        public AdoRestHttpClient(Uri baseUrl, VssCredentials credentials) : base(baseUrl, credentials)
+        {
+            this.baseUrl = baseUrl;
+        }
+
+        public AdoRestHttpClient(Uri baseUrl, VssCredentials credentials, VssHttpRequestSettings settings) : base(baseUrl, credentials, settings)
+        {
+            this.baseUrl = baseUrl;
+        }
+
+        public AdoRestHttpClient(Uri baseUrl, VssCredentials credentials, params DelegatingHandler[] handlers) : base(baseUrl, credentials, handlers)
+
+        {
+            this.baseUrl = baseUrl;
+        }
+
+        public AdoRestHttpClient(Uri baseUrl, HttpMessageHandler pipeline, bool disposeHandler) : base(baseUrl, pipeline, disposeHandler)
+
+        {
+            this.baseUrl = baseUrl;
+        }
+
+        public AdoRestHttpClient(Uri baseUrl, VssCredentials credentials, VssHttpRequestSettings settings, params DelegatingHandler[] handlers) : base(baseUrl, credentials, settings, handlers)
+
+        {
+            this.baseUrl = baseUrl;
+        }
+
+        public HttpClient getHttpClient()
```

Comment:
Pascal case for method names - `GetHttpClient` & `GetOrganizationUrl`

Reply:
Fixed